document/src/main/java/com/ritense/document/web/rest/DocumentDefinitionResource.java

```diff
@@ -58,11 +58,11 @@ ResponseEntity<Page<? extends DocumentDefinition>> getDocumentDefinitionsForMana
     @GetMapping("/v1/document-definition/open/count")
     ResponseEntity<List<UnassignedDocumentCountDto>> getUnassignedDocumentCount();
 
-    @PostMapping(value = "/v1/document-definition", consumes = APPLICATION_JSON_VALUE)
```

Comment:
I think we're supposed to keep supporting old deprecated endpoints for 1 year

Reply:
fixed